packages/ember-glimmer/lib/component-managers/custom.ts

```diff
@@ -221,7 +223,9 @@ export default class CustomComponentManager<ComponentInstance>
   }
 
   didRenderLayout(state: CustomComponentState<ComponentInstance>, bounds: VMBounds) {
-    state.delegate.didRenderLayout(state.component, new Bounds(bounds));
+    if (GLIMMER_COMPONENT_MANAGER_BOUNDS) {
+      state.delegate.didRenderLayout(state.component, new Bounds(bounds));
```

Comment:
One of the feedback from the RFC was that we should actually pass the `state` through instead of just the component. Can you update both hooks to pass it?

Reply:
This file is just confusing. state.component _is_ the userland “bucket”.

Reply:
Done.